frontend/store/auth.js

```diff
@@ -46,13 +53,16 @@ export const actions = {
       commit('setAuthenticated', true)
       commit('setUsername', user.username)
       commit('setUserId', user.id)
+      commit('setIsStaff', false)
```

Comment:
Is `user.isStaff` correct?
Even if the user is staff, the create button is hidden:
![](https://i.gyazo.com/6e38c573b93d5be15f70f1e646e1e0f5.png)

Reply:
fixed. https://github.com/doccano/doccano/pull/1453/commits/a6397eb5ec0071e544fd470a0eb778d17d8bfb3e